dashboard/app/serializers/api/v1/pd/application_quick_view_serializer.rb

```diff
@@ -0,0 +1,15 @@
+class Api::V1::Pd::ApplicationQuickViewSerializer < ActiveModel::Serializer
+  attributes :id, :created_at, :name, :district_name, :school_name, :status
+
+  def name
+    "#{object.sanitize_form_data_hash[:first_name]} #{object.sanitize_form_data_hash[:last_name]}"
+  end
+
+  def school_name
+    object.user.school_info.try(:effective_school_name).try(:titleize) || ''
+  end
+
+  def district_name
+    object.user.school_info.try(:effective_school_district_name).try(:titleize) || ''
+  end
```

Comment:
If we expect these to be accessed commonly, they should be exposed on the model (with tests), and then you won't need to add custom methods here.